src/components/client/TreasuryPage.tsx

```diff
@@ -65,6 +65,7 @@ const TreasuryPage: NextPage = () => {
       </Flex>
       <TreasuryGraphTable
         setTreasuryValue={(tValue: number) => setTreasuryValue(tValue)}
+        hasExternalLink={true}
```

Comment:
I don't think you need to ``hasExternalLink`` like this, you can properly extend this component ``PageHeader`` in ``PageHeader.tsx``, then you pass the url.

```suggestion
        <PageHeader ExternalLink={//link to site} />
```

Reply:
Thanks for the feedback.